packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -141,40 +131,39 @@ contract FundingRateApplier is Lockable {
             currentTime,
             paymentPeriod,
             _latestFundingRatePerSecond.rawValue,
-            effectiveFundingRateForPeriod.rawValue
+            periodRate.rawValue
         );
 
         lastUpdateTime = currentTime;
     }
 
     function _calculateEffectiveFundingRate(
         uint256 paymentPeriodSeconds,
-        FixedPoint.Unsigned memory fundingRatePerSecond,
-        FixedPoint.Unsigned memory feeMultiplier
-    ) internal pure returns (FixedPoint.Unsigned memory, FixedPoint.Unsigned memory) {
+        FixedPoint.Signed memory fundingRatePerSecond,
+        FixedPoint.Unsigned memory currentCumulativeFundingRateMultiplier
+    )
+        internal
+        pure
+        returns (FixedPoint.Unsigned memory newCumulativeFundingRateMultiplier, FixedPoint.Signed memory periodRate)
```

Comment:
+1 for making these named return variables.

Reply:
It was only to save bytecode :)

But I agree it also makes it more readable.